### ISSUE TITLE
add monitoring functionality for station management

### ISSUE THREAD

**Issue by tomkralidis** [wmo-im/wis2box#329]
## User story
As a wis2box user, I want to monitor the state of data that are regularly transmitted, so that I can take corrective action to address data outages

## Acceptance criteria

- [ ] prometheus workflow(s) to collect metrics
- [ ] dashboard (wis2box-ui-admin/grafana) provides overview of reduced and/or failing data/station outages

## Definition of done

- [ ] unit/functional tests added/updated
- [ ] documentation updated
- [ ] live demonstration provided

**Comment by maaikelimper** [comment]
Update metrics went in, a new metric was added:
```
failure_descr_wsi_total = Counter('wis2box_failure_detail_total',
                                    'Total failed actions sent by wis2box, by description and WSI', # noqa
                                    ["description", "WSI"])
```

I'm not sure this will be sufficient and we need a dashboard to display data from this metric

**Comment by tomkralidis** [comment]
2023-01-17: consider adding email notification capability.  Needs investigation on SMTP setup.

**Comment by maaikelimper** [comment]
Opened
https://github.com/wmo-im/csv2bufr/issues/89
to help provide information to wis2box about which station is failing to produce data

**Comment by tomkralidis** [comment]
Implement custom status block in csv2bufr/synop2bufr and wis2box internal plugins.

@david-i-berry @RoryPTB

**Comment by tomkralidis** [comment]
@maaikelimper to add failure message publishing based on result using csv2bufr master.

**Comment by tomkralidis** [comment]
Addressed in #416
